primitives/core/src/network.rs

```diff
@@ -78,6 +78,24 @@ impl Web3Network {
 	}
 }
 
+impl Web3Network {
+	/// Achainable server support lowercase ONLY.
```

Comment:
I would suggest putting `Achainable` related code into `Achainable` context, probably `achainable.rs`.

What about adding this function to `achainable.rs` ?

```
pub fn web3_network_to_chain(network: &Web3Network) -> &'static str {
	match network {
		Web3Network::Polkadot => "polkadot",
		Web3Network::Kusama => "kusama",
		Web3Network::Litentry => "litentry",
		Web3Network::Litmus => "litmus",
		Web3Network::LitentryRococo => "litentry_rococo",
		Web3Network::Khala => "khala",
		Web3Network::SubstrateTestnet => "substrate_testnet",
		Web3Network::Ethereum => "ethereum",
		Web3Network::Polygon => "polygon",
		Web3Network::BSC => "bsc",
	}
}
```